pipelines/nextflow/README.md

```diff
@@ -12,12 +18,16 @@ This is an example of how the tree folder for Nextflow pipelines has been agreed
         ├── conf
         │   └── base.config
         ├── modules
```

Comment:
I made some slight changes to displayed folder structure hope thats ok

Reply:
ok. thanks!